main.py: main()関数に不正なreturn文がある [func-returns-value]

## 概要

mypy が `main.py` で以下の実バグを検出している。

```
src/lorairo/main.py:278: error: No return value expected  [return-value]
src/lorairo/main.py:282: error: "main" does not return a value (it only ever returns None)  [func-returns-value]
```

## 詳細

`main()` 関数が `None` しか返さないにもかかわらず、`return` 文で値を返そうとしている（または戻り値のある関数として呼び出した結果を使用している）。

## 影響

アプリケーション起動処理に関わるため、エントリーポイントの動作が意図通りでない可能性がある。

## 関連

- Issue #75 (mypy 型エラー段階的修正) の調査中に発見
- `src/lorairo/main.py` 行 278, 282

--- a/src/lorairo/main.py
+++ b/src/lorairo/main.py
@@ -180,7 +180,7 @@ def parse_arguments() -> argparse.Namespace:
     return parser.parse_args()
 
 
-def main() -> None:
+def main() -> int:
     """メイン実行関数"""
     # コマンドライン引数解析
     args = parse_arguments()

